setup.py

```diff
@@ -12,7 +12,7 @@
     packages=find_packages(exclude=['tests', 'tests.*']),
     include_package_data=True,
     install_requires=[
-        'six==1.8.0'
+        'six==1.10.0'
```

Comment:
Had to make this change for the build to pass. Build was failing with the six version on Python 3.4.6